src/index.html

```diff
@@ -16,17 +16,32 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <form
+      class="search-bar big"
+      data-qa="big"
+    >
+      <svg class="icon__big"></svg>
+      <input
+        class="input__big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“ "
```

Comment:
Avoid using spaces in the placeholder text. Consider using a non-breaking space or adjusting the text to avoid spaces.